src/System.Security.Cryptography.Cng/tests/ImportExportTests.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Win32.SafeHandles;
+using Test.Cryptography;
+using Xunit;
+
+namespace System.Security.Cryptography.Cng.Tests
+{
+    public static class ImportExportTests
+    {
+        [Fact]
+        [PlatformSpecific(PlatformID.Windows)]
+        public static void TestImportExport1()
+        {
+
+            byte[] reExported;
+            using (CngKey key = CngKey.Import(TestData.Key1, CngKeyBlobFormat.GenericPublicBlob))
+            {
+                SafeNCryptKeyHandle keyHandle = key.Handle;
+                Assert.NotNull(keyHandle);
+                keyHandle.Dispose();
+                Assert.Equal(CngAlgorithm.ECDiffieHellmanP256, key.Algorithm);
+                Assert.Equal(CngAlgorithmGroup.ECDiffieHellman, key.AlgorithmGroup);
+                Assert.Equal(CngExportPolicies.None, key.ExportPolicy);
+                Assert.Equal(true, key.IsEphemeral);
+                Assert.Equal(false, key.IsMachineKey);
+                Assert.Equal(null, key.KeyName);
+                Assert.Equal(0x100, key.KeySize);
+                Assert.Equal(CngKeyUsages.AllUsages, key.KeyUsage);
+                Assert.Equal(IntPtr.Zero, key.ParentWindowHandle);
+                Assert.Equal(CngProvider.MicrosoftSoftwareKeyStorageProvider, key.Provider);
+
+                CngUIPolicy policy = key.UIPolicy;
+                Assert.Equal(null, policy.CreationTitle);
```

Comment:
The test is checking a bunch of properties with a mix of null and non-null values so I'd rather use Assert.Equal() for all of them for consistency.